.buildkite/pipelines/lucene-snapshot/build-snapshot.yml

```diff
@@ -0,0 +1,20 @@
+steps:
+  - trigger: apache-lucene-build-snapshot
+    label: Trigger pipeline to build lucene snapshot
+    key: lucene-build
+    if: build.env("LUCENE_BUILD_ID") == null || build.env("LUCENE_BUILD_ID") == ""
+  - wait
+  - label: Upload and update lucene snapshot
+    command: .buildkite/scripts/lucene-snapshot/upload-snapshot.sh
+    timeout_in_minutes: 120
```

Comment:
We can probably reduce this timeout now that we're just uploading an existing artifact.